CommonAssemblyInfo.cs

```diff
@@ -1,4 +1,4 @@
 using System.Reflection;
 
-[assembly: AssemblyVersion("6.5.0")]
-[assembly: AssemblyFileVersion("6.5.0")]
+[assembly: AssemblyVersion("6.90.0")]
```

Comment:
Nevermind noticed that it comes from the merge. Can you retarget this to rxui7-master?